foreman/foreman.spec

```diff
@@ -737,6 +745,8 @@ webpack.js --bail --config config/webpack.config.js
 %{scl_rake} apipie:cache RAILS_ENV=production cache_part=resources --trace
 rm config/database.yml config/settings.yaml
 
+sed -ri '1sX(/usr/bin/ruby|/usr/bin/env ruby)X/usr/bin/%{?scl:%{scl_prefix}}rubyX' bin/dynflow-executor
```

Comment:
I think this line is redundant because of https://github.com/theforeman/foreman-packaging/blob/22efa6c9e1b22f82aff3f62b6960bd99d2ce89fd/foreman/foreman.spec#L816-L825

Reply:
Correct, will remove it